src/commonMain/kotlin/software/momento/kotlin/sdk/responses/cache/control/CacheInfo.kt

```diff
@@ -0,0 +1,18 @@
+package software.momento.kotlin.sdk.responses.cache.control
+/** Information about a cache.  */
+public class CacheInfo
+
+/**
```

Comment:
The `@param` can be an `@property` in the KDoc at the top of the class, and then this block isn't needed.